pkgs/development/libraries/gstreamer/core/default.nix

```diff
@@ -30,7 +30,7 @@ stdenv.mkDerivation rec {
     # - https://github.com/NixOS/nixpkgs/pull/98767
     # - https://github.com/NixOS/nixpkgs/issues/98769#issuecomment-702296551
   ];
-  outputBin = "dev";
+  outputBin = "out";
```

Comment:
As mentioned on the commit, I would not put them to out. I would remove `outputBin` and add `bin` as the first output to `outputs`.